flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
since it is using dotnet and the runtime exists for aarch64, is there a reason to limit to x86_64?